examples/example_mineral_lstm.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (C) 2017-2018 COAL Developers
+#
+# This program is free software; you can redistribute it and/or 
+# modify it under the terms of the GNU General Public License 
+# as published by the Free Software Foundation; version 2.
+#
+# This program is distributed in the hope that it will be useful, 
+# but WITHOUT ANY WARRANTY; without even the implied warranty 
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public 
+# License along with this program; if not, write to the Free 
+# Software Foundation, Inc., 51 Franklin Street, Fifth 
+# Floor, Boston, MA 02110-1301, USA.
+# encoding: utf-8
+
+'''
+example_mineral_lstm -- an example script which demonstrates COAL mineral classification
+                        using a binary classifier based on Recurrent Neural Network
+
+example_mineral_lstm provides a CLI which demonstrates how the COAL Mineral Classification 
+API provides methods for generating visible-light and mineral classified images. 
+Mineral classification can take hours to days depending on the size of the spectral 
+library and the available computing resources, so running a script in the background 
+is recommended. More reading an this example can be seen at 
+https://capstone-coal.github.io/docs#usage
+
+@author:     COAL Developers
+
+@copyright:  Copyright (C) 2017-2018 COAL Developers
+
+@license:    GNU General Public License version 2
+
+@contact:    coal-capstone@googlegroups.com
+'''
+
+import sys
+import os
+from sys import path
+from os import getcwd
+import inspect
+
+from argparse import ArgumentParser
+from argparse import RawDescriptionHelpFormatter
+
+import logging
+
+import sys
+import os
+import pycoal
+sys.path.insert(0, '../pycoal')
+import mineral
+import mining
+import environment
+
+__all__ = []
+
+DEBUG = 1
+TESTRUN = 0
+PROFILE = 0
+
+
+input_filename = 'avng.jpl.nasa.gov/AVNG_2015_data_distribution/L2/ang20150420t182050_rfl_v1e/ang20150420t182050_corr_v1e_img.hdr'
+model_filename='../pycoal/tests/nn_models/lstm_Schwertmannite.h5'
```

Comment:
This model need to be included in the pull request. Please include it!

Reply:
@lewismc, My apologies, I forgot to include the model's folder into the PR. I'm going to include it right away!